### DISCUSSION TITLE
How to install/uninstall in windows 11

### DISCUSSION THREAD

**Discussion by sujfuy** [mifi/lossless-cut#2090] [General]
I was trying to update to the latest version for windows using the zip file, but windows will not let me extract to the program folder.  That is currently where the old version is and windows will not let me delete it. I can extract to the download folder and run the program, but that is not desirable.

**Comment by mifi** [comment]
can you show some screenshots of the errors you're getting? sounds like you're having some permission issues, maybe you need to run as admin when extracting to the program files folder?
